logger/src/lib.rs

```diff
@@ -185,7 +185,7 @@ lazy_static! {
     /// Static instance used for handling human-readable logs.
     ///
     pub static ref LOGGER: &'static Logger = {
-        set_logger(_LOGGER_INNER.deref()).unwrap();
+        set_logger(_LOGGER_INNER.deref()).expect("Failed to set logger: already initialized");
```

Comment:
Seems that the error returned by the `set_logger ` is this one:
`static SET_LOGGER_ERROR: &'static str = "attempted to set a logger after the logging system \
                                         was already initialized";`
Since expect will append the message returned by the `set_looger` function, I say we only leave the part "Failed to set logger" in our error message.